.mockery.yaml

```diff
@@ -0,0 +1,45 @@
+with-expecter: true
+dir: "mocks"
+filename: "{{.InterfaceName | lower }}.generated.go"
+mockname: "{{.InterfaceName}}"
+outpkg: "mocks"
+packages:
+  github.com/ethereum/go-ethereum/event:
+    config:
+    interfaces:
+      Subscription:
+        config:
+  github.com/0xPolygon/cdk-data-availability/db:
+    config:
+    interfaces:
+      DB:
+        config:
+      Tx:
+        config:
+  github.com/0xPolygon/cdk-data-availability/client:
+    config:
+    interfaces:
+      ClientFactory:
+        config:
```

Comment:
Can you provide the filename config here as well, to be `client_factory.generated.go`, instead of `clientfactory.generated.go`?

Reply:
Good point